doc/examples/filters/plot_hysteresis.py

```diff
@@ -0,0 +1,50 @@
+"""
+=======================
+Hysteresis thresholding
+=======================
+
+*Hysteresis* is the lagging of an effect — a kind of inertia. In the
+context of thresholding, it means that areas above some *low* threshold
+are considered to be above the threshold *if* they are also connected
+to areas above a higher, more stringent, threshold. They can thus be
+seen as continuations of these high-confidence areas.
+
+Below, we compare normal thresholding to hysteresis thresholding.
+Notice how hysteresis allows one to ignore "noise" outside of the coin
+edges.
+"""
+
+import matplotlib.pyplot as plt
+from skimage import data, filters
+
+fig, ax = plt.subplots(nrows=2, ncols=2,
+                       subplot_kw={'adjustable': 'box-forced'})
+
+image = data.coins()
+edges = filters.sobel(image)
+
+low = 0.05
```

Comment:
The point depends on downstream analysis but ideally, yes, you'd want only the contour of each coin. But no threshold combo can do that. In my example, you had perfect recall for those edges, and you removed some of the cruft from outside the coins. In your example, you dropped a couple of contours *but* you get much much better precision, which makes it much more visually striking. I've updated the example to use your suggested threshold.